src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/AnalyzeCovariates.java

```diff
@@ -84,62 +82,62 @@
  *
  * <h4>Plot a single recalibration table</h4>
  * <pre>
- *      ./gatk-launch AnalyzeCovariates \
- *      -R reference.fasta \
- *      -bqsr recal1.table \
- *      -plots AnalyzeCovariates.pdf
+ *   gatk AnalyzeCovariates \
+ *     -R reference.fasta \
+ *     -bqsr recal1.table \
+ *     -plots AnalyzeCovariates.pdf
  * </pre>
  *
  * <h4>Plot "before" (first pass) and "after" (second pass) recalibration tables to compare them</h4>
  * <pre>
- *      ./gatk-launch AnalyzeCovariates \
- *      -R reference.fasta \
- *      -before recal1.table \
- *      -after recal2.table \
- *      -plots AnalyzeCovariates.pdf
+ *   gatk AnalyzeCovariates \
+ *     -R reference.fasta \
+ *     -before recal1.table \
+ *     -after recal2.table \
+ *     -plots AnalyzeCovariates.pdf
  * </pre>
  *
  * <h4>Plot up to three recalibration tables for comparison</h4>
  * <pre>
- *      ./gatk-launch AnalyzeCovariates \
- *      -R reference.fasta \
- *      -ignoreLMT \
- *      -bqsr recal1.table \
- *      -before recal2.table \
- *      -after recal3.table \
- *      -plots AnalyzeCovariates.pdf
+ *   gatk AnalyzeCovariates \
+ *     -R reference.fasta \
+ *     -ignoreLMT \
+ *     -bqsr recal1.table \
+ *     -before recal2.table \
+ *     -after recal3.table \
+ *     -plots AnalyzeCovariates.pdf
  * </pre>
  *
  * <h4>Full BQSR quality assessment pipeline</h4>
  * <p>Generate the first pass recalibration table file</p>
  * <pre>
- *      ./gatk-launch BaseRecalibrator \
- *      -R reference.fasta \
- *      -I input.bam \
- *      -knownSites my-trusted-snps.vcf \
- *      -knownSites my-trusted-indels.vcf \
- *      -O recal1.table
+ *   gatk BaseRecalibrator \
```

Comment:
Let's also remove the entirety of the `Full BQSR quality assessment pipeline` section. Such an overview belongs in a best practice tutorial and not in the tool doc. And actually, the tools have changed such that now some of these actually will error.

Reply:
Done